src/drivers/uavcan/sensors/flow.cpp

```diff
@@ -67,6 +67,9 @@ UavcanFlowBridge::flow_sub_cb(const uavcan::ReceivedDataStructure<com::hex::equi
 {
 	optical_flow_s flow{};
 
+	// We're only given an 8 bit field for sensor ID; just use the UAVCAN node ID
+	flow.sensor_id = (uint8_t)(msg.getSrcNodeID().get() & 0xFF);
```

Comment:
cast is redundant, `msg.getSrcNodeID().get()` returns `uint8_t` and `flow.sensor_id` is `uint8_t`